src/station_gui.cpp

```diff
@@ -445,7 +445,14 @@ class CompanyStationsWindow : public Window
 				bool rtl = _current_text_dir == TD_RTL;
 				int max = std::min<size_t>(this->vscroll->GetPosition() + this->vscroll->GetCapacity(), this->stations.size());
 				int y = r.top + WD_FRAMERECT_TOP;
-				for (int i = this->vscroll->GetPosition(); i < max; ++i) { // do until max number of stations of owner
+				uint line_height = this->GetWidget<NWidgetBase>(widget)->resize_y;
+				/* Spacing between station name and first rating graph. */
+				uint text_spacing = ScaleFontTrad(5);
+				/* Spacing between additional rating graphs. */
+				uint rating_spacing = ScaleFontTrad(4);
+
+				for (int i = this->vscroll->GetPosition(); i < max; ++i)
+				{ // do until max number of stations of owner
```

Comment:
```suggestion
				// do until max number of stations of owner
				for (int i = this->vscroll->GetPosition(); i < max; ++i) {
```

Reply:
Have reverted this change, unintentional autoreformat!